images/macos/provision/configuration/finalize-vm.sh

```diff
@@ -17,4 +17,9 @@ cp $HOME/image-generation/output/software-report/systeminfo.txt $HOME/image-gene
 npm cache clean --force
 
 # Clean up temporary directories
-rm -rf ~/utils ~/image-generation
\ No newline at end of file
+rm -rf ~/utils ~/image-generation
+
+# Erase all indexes and wait until the rebuilding process ends,
+# for now there is no way to get status of indexing process, it takes around 3 minutes to accomplish
+sudo mdutil -E /
+sleep 180
```

Comment:
I don't think that it is good idea to use hardcoded 3 minutes.
What if we add more tools tomorrow, and it will take 4 minutes?
What if VM performance degrade tomorrow and it will take 10 minutes?